diagnostic.md

```diff
@@ -14,7 +14,7 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$('li').val()
```

Comment:
`$('li');` is enough to select all `<li>` elements.  `.val()` is typically used to get and set the values of input fields.  From `<ol>`, two possible ways to select `<li>` elements are `$('ol > li');` and `$('ol').children();`.  Some other possible ways are `$('li');`, `$('body').find('li');`, and `$('body').children('ol').children('li');`.